earth_enterprise/src/server/mod_fdb/fusiondbservice.cpp

```diff
@@ -103,6 +103,8 @@ int FusionDbService::ProcessFusionDbRequest(
   bool is_time_machine = false;
 
   if (cmd_or_path == "dbRoot.v5") {
+    ap_log_rerror(APLOG_MARK, APLOG_DEBUG, 0, r,
```

Comment:
Lint: Do we want to log every dbRoot read?

Reply:
Actually, I think so - they're much more infrequent than you'd think, AFAICT, which I could only discern by using these debug log statements!